eng/pipelines/templates/steps/resolve-package-targeting.yml

```
@@ -13,36 +13,12 @@ steps:
   # whether we are running within a PR build or not, we need to walk the package properties and save them to
   # variable $(TargetingString) so that every other check and accept it as an argument.
   - pwsh: |
-      $setting = "${{ parameters.BuildTargetingString }}"
+      $IncludeIndirect = [bool]::Parse("${{ parameters.IncludeIndirect }}")
```

```suggestion
      $IncludeIndirect = $${{ parameters.IncludeIndirect }}
```

FYI another option, either works.

---

I'm not taking it just because I don't want to add another commit. It'll work just the same. I'll keep remember this for certain 👍